packages/ui/src/components/accordion/accordion.vue

```diff
@@ -90,29 +88,25 @@
 
   const classes = computed(() => {
     return {
-      'acv-accordion': true,
       [`acv-accordion--size-${size}`]: size,
     };
   });
 </script>
 
 <template>
   <div
-    class="acv accordion"
```

Comment:
we must prohibit using unprefixed classes. They can be mistakenly corrupted by library user